src/functions/Output.ps1

```diff
@@ -553,7 +542,13 @@ function Get-WriteScreenPlugin ($Verbosity) {
 
 
     $p.RunStart = {
-        & $SafeCommands["Write-Host"] -ForegroundColor Magenta "Running tests."
+        param ($Context)
+
+        $testsToRun = 0
+        foreach ($test in @(View-Flat -Block $Context.Blocks)) {
+            if ($test.ShouldRun) { $testsToRun++ }
+        }
+        & $SafeCommands["Write-Host"] -ForegroundColor Magenta "Running $testsToRun tests."
```

Comment:
Rephrase? "Running x tests in total" maybe? Might feel weird currently considering the next line is "Running tests from <file number 1>"

Reply:
Maybe the filter output should move after discovery? Then:
Discovery found 16 tests in 34 ms. (instead of Discovery finished)
Filtering tests with filter: ....
Filters selected 4 tests to run.

Reply:
I'm guessing that includes the list of active filters. It would be more natural to delay all filter-related messages in the output until they're actually processed. This is a "known issue" from https://github.com/pester/Pester/pull/1727#issue-509026272 as we don't have a plugin-step between discovery and filter/post-process. Would have to do host-output in the middle of the code (or implement something like DiscoveryPostProcessStart-step).

Currently we can update the summary in `DiscoveryEnd`-step like below and then keep the "Running tests." message after to indicate Run-phase has begun.
```
Discovery found 16 tests in 34 ms. 
Filters selected 4 tests to run.
```

Reply:
I see. How about just postponing writing the filter out, and doing it after we are done filtering in DiscoveryEnd. 

<actual filtering is happening>
Filtered tests with filters: ....
Filters selected 4 tests to run.
Discovery finished in 34 ms.